README.md

```diff
@@ -1,143 +1,198 @@
-
-Juvix's compiler architecture is purpose-built from the ground up for the particular requirements and economic trade-offs
-of the smart contract use case — it prioritises behavioural verifiability, semantic precision, and output code efficiency over compilation speed,
-syntactical familiarity, and backwards compatibility with existing blockchain virtual machines.
-
-For more design details, see [the language
-reference](./doc/reference/language-reference.pdf).
-
-<!--## Screenshots TODO -->
-<!-- Include logo/demo screenshot with labels.
-- vscode screenshot
-- eg contracts -->
-
-## Caveats
-
-This is pre-alpha software released for experimentation & research purposes only.
-
-Do not expect API stability. Expect bugs. Expect divergence from canonical protocol implementations.
-
-Formal verification of various properties of the Juvix language & compiler in Agda is [in progress](experimental/qtt-agda) but not yet complete.
-
-No warranty is provided or implied.
-
-## Installation
-
-<!-- ### Requirements
-
-The following are required to build Juvix:
-
-- [libff](https://github.com/scipr-lab/libff)
-- [libsecp256k1](https://github.com/bitcoin-core/secp256k1)
-- [Openssl Libssl API](https://wiki.openssl.org/index.php/Libssl_API) -->
-
-
-[Stack](https://haskellstack.org) required. 8GB RAM required for `stack` installation.
+[![stability-experimental](https://img.shields.io/badge/stability-experimental-orange.svg)](https://github.com/mkenney/software-guides/blob/master/STABILITY-BADGES.md#experimental)
+![GitHub release (latest SemVer including pre-releases)](https://img.shields.io/github/v/release/anoma/juvix?include_prereleases)
+====
+
+Description
+-----------
+
+<img align="right" width="300" height="300" src="Juvix_logo.png">
+
+Juvix is a dependently functional programming language for writing
+efficient formally-verified smart contracts, which can be
+deployed to various distributed ledgers. Juvix addresses many issues that we have
+experienced while trying to write and deploy decentralised
+applications present in the ecosystem of smart-contracts: 
+- the difficulty of
+adequate program verification, 
+- the ceiling of compositional
+complexity, 
+- the illegibility of execution costs, and
+- the lock-in to
+particular backends.
+
+The Juvix compiler synthesises a high-level frontend syntax with
+support for dependent-linearly types and several other cutting-edge
+research ideas from programming language design and type theory. For
+more details, see [the Juvix language
+reference WIP](./doc/reference/language-reference.pdf).
+
+Prerequisites
+--------------
+
+* Install [Stack](https://haskellstack.org):
 
   - For Ubuntu        : `apt install stack`
   - For Debian        : `apt install haskell-stack`
   - For Arch Linux    : `pacman -S stack`
   - For macOS : `brew install haskell-stack`
```

Comment:
misaligned?

Reply:
Good catch!

Reply:
Oh wait! do you mean the "For Ubuntu        : " thing or what exactly.

Reply:
macOS was misaligned.